mods/cnc/tilesets/temperat.yaml

```diff
@@ -449,8 +449,8 @@ Templates:
 		Size: 2,2
 		Category: Cliffs
 		Tiles:
-			0: Clear
-			1: Clear
+			0: Rock
```

Comment:
I think this (and below) should remain passable.
![tdcliff3](https://cloud.githubusercontent.com/assets/7704140/12011129/9d332e96-acc2-11e5-92da-2118d3a56b6c.png)